java/src/jmri/jmrit/powerpanel/PowerPane.java

```diff
@@ -199,6 +205,29 @@ public void idleButtonPushed() {
         }
     }
 
+    /**
+     * Get Minimum width for the current power status JLabel.
+     * On / off are unlikely to be long strings so we check Idle / Unknown.
```

Comment:
Not true. See the [PowerPanelBundle_nl.properties](https://github.com/JMRI/JMRI/blob/master/java/src/jmri/jmrit/powerpanel/PowerPanelBundle_nl.properties) file in this folder.

```
StatusOn      = Ingeschakeld
StatusOff     = Uitgeschakeld
StatusUnknown = Onbekend
StatusIdle    = Idle
```
StatusIdle is not translated so the English word will be used by JMRI.